scripts/make_port_status.py

```diff
@@ -145,7 +146,7 @@ def get_mathlib4_module_commit_info(contents):
         f = subprocess.run(
             ['git', 'cat-file', 'blob', f'port-status-pull/{num}:{l}'],
             capture_output=True)
-        import_, repo, commit = get_mathlib4_module_commit_info(f.stdout.decode())
+        import_, repo, commit = get_mathlib4_module_commit_info(f.stdout.decode(errors='replace'))
```

Comment:
I think this should be
```suggestion
        import_, repo, commit = get_mathlib4_module_commit_info(f.stdout.decode(encoding='utf8'))
```

Reply:
Let me try this.

Reply:
That does not work. It again fails with the error I encountered earlier:
`UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8c in position 19: invalid start byte`.

Reply:
It fails on `port-status-pull/6352:lakefile.olean`.

Reply:
In that case, I think you should combine your version with my version. Also clearly #6352 is bad!

Reply:
I did, see the updated version.